crates/node/src/command.rs

```diff
@@ -218,9 +205,81 @@ pub fn run() -> sc_cli::Result<()> {
             cli.run.run_cmd.network_params.node_key_params.node_key_file =
                 Some((madara_path.clone() + "/p2p-key.ed25519").into());
 
+            if cli.run.genesis_url.is_some() {
+                // can't copy extra genesis-assets atm
+                // we can reuse #982 to create the standard to fetch relevant files
+                let copied = utils::fetch_from_url(
+                    cli.run.genesis_url.clone().unwrap(),
+                    madara_path.clone() + "/configs/genesis-assets",
+                );
+
+                if !copied {
+                    return Err("Failed to fetch genesis assets".into());
+                }
+            } else {
+                // TODO confirm with the CI that we are fetching all and fetch dynamically
+                // Issue #982
+                for file in constants::GENESIS_ASSETS_FILES {
+                    let src_path: String = utils::get_project_path() + "/configs/genesis-assets/" + file;
+                    let mut copied: bool =
+                        utils::copy_from_filesystem(src_path, madara_path.clone() + "/genesis-assets");
```

Comment:
you could replace the return from `copy_from_filesystem` by a `Result<(), String>` and then have something like 
```
let res = utils::copy_from_filesystem(src_path, madara_path.clone() + "/genesis-assets");
if res.is_err() {
    utils::fetch_from_url(
        constants::GENESIS_ASSETS_URL.to_string() + file,
        madara_path.clone() + "/genesis-assets",
    )?;
}
```